nservicebus/msmq/scalability-and-ha/sender-side-distribution.md

```diff
@@ -0,0 +1,120 @@
+
+There are many different options to consider when deploying routing configuration.
+
+* Many endpoints can be configured to use one centralized mapping file on a network drive accessible by all, creating a single file that describes how messages flow across an entire system. Any given endpoint will not care if the file contains information for endpoints it does not need.
+* The mapping file can be kept in a centralized location and replicated out to many servers on demand, allowing each endpoint to read the file from a location on the local disk.
+* Each endpoint can keep its own routing file containing only information for the endpoints it needs to know about, which can be deployed in the same directory as the endpoint binaries and only modified as part of a controlled deployment process.
+
+You can change the following default settings:
+ 
+ 
+### RefreshInterval
+
+Specifies the interval between route data refresh attempts.
+
+Default: 30 seconds
+
+snippet: Routing-FileBased-RefreshInterval
+
+
+### FilePath
+
+Specifies the path and file name of the instance mapping file. This can be a realtive or an absolute file path. Relative file paths are resolved from `AppDomain.BaseDirectoy`.
+
+Default: `instance-mapping.xml`
+
+snippet: Routing-FileBased-FilePath
+
+
+## The file
+
+The routing file is a simple XML file that has to be located either on local hard drive or a network drive. It is processed before the endpoint starts up and then re-processed in regular intervals so the changes in the document are reflected in the behavior of NServiceBus automatically (with slight delay caused by caching of routes). If the document is not present in its configured location when endpoint starts up, NServiceBus will not search again for the file during runtime. If it is deleted or corrupted while the endpoint is already running, NServiceBus will continue to run normally with last successfully read routes. 
+
+
+### Examples of routing files
+
+Following are examples of instance mapping configurations for the given sample routing:
+
+snippet:Routing-FileBased-Config
+
+
+#### Machine name
+
+The mapping file contains a `Machine` attribute which needs to be used when the receiving endpoint runs on a different machine. If no machine attribute is specified, MSMQ assumes the receiving endpoint is located on the same machine.
+
+snippet:Routing-FileBased-MSMQ
+
+
+#### Instance discriminator
+
+The instance discriminator attribute can be used if multiple uniquely addressable endpoints run on the same machine and messages should be sent to their unique input queue instead of the shared input queue.
+
+snippet:Routing-FileBased-Broker
```

Comment:
I'm a bit confused, because seems the parser is parsing what fits the non-msmq snippet (see here https://github.com/Particular/NServiceBus/blob/release-6.0.0/src/NServiceBus.Core/Routing/FileBasedDynamicRouting/FileRoutingTableParser.cs#L35). 
Am I looking at the wrong thing? Or did the MSMQ file structure changed and that's for parsing msmq settings only?